packages/cli/package.json

```diff
@@ -316,10 +316,10 @@
     "postpublish": "rm -f oclif.manifest.json",
     "prepack": "yarn run build && oclif manifest",
     "pretest": "tsc -p test --noEmit && cd ../.. && yarn build",
-    "test": "yarn pretest && nyc mocha --forbid-only \"test/**/*.unit.test.ts\"",
-    "test:acceptance": "mocha --forbid-only \"test/**/*.acceptance.test.ts\" && node ./bin/bats-test-runner",
-    "test:integration": "mocha --forbid-only \"test/**/*.integration.test.ts\"",
-    "test:smoke": "mocha --forbid-only \"test/**/smoke.acceptance.test.ts\"",
+    "test": "yarn pretest && nyc mocha --forbid-only \"test/**/*.unit.test.ts\" && yarn posttest",
+    "test:acceptance": "yarn pretest && mocha --forbid-only \"test/**/*.acceptance.test.ts\" && node ./bin/bats-test-runner",
```

Comment:
We already run `yarn lerna run prepack` in the ci prior to running the acceptance and smoke tests. https://github.com/heroku/cli/blob/74f8165a5498752385859f84c3e89116f05c023e/.github/workflows/ci.yml#L62

Reply:
Good catch